node-gui/src/main_window/main_widget/tabs/wallet.rs

```diff
@@ -0,0 +1,555 @@
+            ),
+            SelectedPanel::Stake => view_stake(
+                &node_state.chain_config,
+                account,
+                &self.account_state.stake_amount,
+            ),
+        };
+
+        let body = Scrollable::new(container(body).padding(10))
+            .width(Length::Fill)
+            .height(Length::Fill);
+
+        let top_panel = container(view_top_panel(
+            &node_state.chain_config,
+            wallet_info,
+            account,
+        ))
+        .height(50)
+        .width(Length::Fill);
+        let left_panel = container(view_left_panel(
+            node_state,
+            wallet_info,
+            self.selected_account,
+            self.selected_panel,
+        ))
+        .width(150);
+
+        let page: Element<'static, WalletMessage> = row![
+            left_panel.width(Length::Fixed(150.0)),
+            vertical_rule(1),
+            column![top_panel, horizontal_rule(1.0), body]
+        ]
+        .width(Length::Fill)
+        .into();
+
+        page.map(|msg| TabsMessage::WalletMessage(self.wallet_id, msg))
+    }
+}
+
+fn view_top_panel(
+    chain_config: &ChainConfig,
+    wallet_info: &WalletInfo,
+    account: &AccountInfo,
+) -> Element<'static, WalletMessage> {
+    let balance = account.balance.get(&Currency::Coin).cloned().unwrap_or(Amount::ZERO);
+    let balance = print_coin_amount(chain_config, balance);
+    let balance = format!("{} Mlt", balance);
+    let balance = Text::new(balance).size(20);
+
+    let password =
+        match wallet_info.encryption {
+            EncryptionState::EnabledLocked => {
+                row![iced::widget::button(Text::new("Unlock")).on_press(WalletMessage::Unlock)]
+            }
+            EncryptionState::EnabledUnlocked => row![
+                iced::widget::button(Text::new("Lock")).on_press(WalletMessage::Lock),
+                iced::widget::button(Text::new("Disable wallet encryption"))
+                    .on_press(WalletMessage::RemovePassword)
+            ],
+            EncryptionState::Disabled => row![iced::widget::button(Text::new("Encrypt wallet"))
+                .on_press(WalletMessage::SetPassword),],
+        }
+        .align_items(Alignment::Center)
+        .spacing(10);
+
+    row![
+        balance,
+        Row::new().width(Length::Fill),
+        password,
+        button(Text::new("Close"))
+            .style(iced::theme::Button::Destructive)
+            .on_press(WalletMessage::Close),
+    ]
+    .width(Length::Fill)
+    .height(Length::Fill)
+    .spacing(10)
+    .padding(10)
+    .align_items(Alignment::Center)
+    .into()
+}
+
+#[derive(Clone, PartialEq, Eq)]
+struct AccountPickItem {
+    account_id: AccountId,
+    name: String,
+}
+
+impl std::fmt::Display for AccountPickItem {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        std::fmt::Display::fmt(&self.name, f)
+    }
+}
+
+fn view_left_panel(
```

Comment:
Is it possible to make the left panel resizable with the mouse?

Reply:
I did not find a better way and implemented it using the `pane_grid` widget.

Reply:
Looking fine. Thanks.